plugins/homepage/server/__init__.py

```diff
@@ -36,14 +38,27 @@ def __init__(self):
         Description('Public url for getting the homepage markdown.')
     )
     def getMarkdown(self, params):
-        return {KEY: self.model('setting').get(KEY)}
+        folder = getOrCreateAssetsFolder()
+        return {
+            KEY: self.model('setting').get(KEY),
+            'folderId': folder['_id']
+        }
 
 
 def validateSettings(event):
     if event.info['key'] == KEY:
         event.preventDefault().stopPropagation()
 
 
+def getOrCreateAssetsFolder():
+    collection = ModelImporter.model('collection').createCollection(
+        NAME, public=False, reuseExisting=True)
+    folder = ModelImporter.model('folder').createFolder(
+        collection, NAME, parentType='collection', public=False,
+        reuseExisting=True)
```

Comment:
Indeed. Make the collection private and the folder public. Then everyone can download from it without seeing it in their listing.